src/util.h

```diff
@@ -150,4 +161,58 @@ static inline double d_gas_aerosol_rxn_rate_constant_d_radius(
                           mean_free_path__m, radius__m, alpha));
 }
 
+/** Calculate the gas-aerosol reaction rate for the continuum regime
```

Comment:
Can we add some sort of citation for the source of this equation?

Reply:
added citation to Tie et al. 2003